pkg/asset/agent/image/oc.go

```diff
@@ -113,14 +121,14 @@ func (r *release) extractFileFromImage(log logrus.FieldLogger, image, file, cach
 		cmd = fmt.Sprintf(templateImageExtract, file, cacheDir, image)
 	}
 
-	log.Infof("extracting %s to %s, %s", file, cacheDir, cmd)
+	log.Debugf("extracting %s to %s, %s", file, cacheDir, cmd)
 	_, err := retry.Do(r.config.MaxTries, r.config.RetryDelay, execute, log, r.executer, pullSecret, cmd)
 	if err != nil {
 		return "", err
 	}
 	// set path
 	path := filepath.Join(cacheDir, file)
-	log.Infof("Successfully extracted %s binary from the release to: %s", file, path)
+	log.Infof("Successfully extracted %s ISO from the release to: %s", file, path)
```

Comment:
Should we be logging the path to the cache dir at INFO level? This seems like more of a debug thing.